packages/generator/tests/conftest.py

```diff
@@ -37,3 +41,19 @@ def discovery_service_process() -> Generator[DiscoveryServiceProcess, None, None
 def pin_map_directory(test_assets_directory: pathlib.Path) -> pathlib.Path:
     """Test fixture that returns the pin map directory."""
     return test_assets_directory / "pin_map"
+
+
+@pytest.fixture(scope="session")
+def create_client() -> Generator[functools.partial[Result], Any, None]:
```

Comment:
The middle type parameter for `Generator` is the "send type". This is only used for coroutines. You don't "send" to this generator, so use `None` for the 2nd and 3rd type parameters.